cmd/stbtrace.py

```diff
@@ -136,5 +136,8 @@
 else:
```

Comment:
Would be good to add an `exit(0)` for the printing script (`-b`) case, otherwise lgtm